catalog/installSystem.sh

```diff
@@ -13,23 +13,51 @@ cd "$SCRIPTDIR/../bin"
 
 install utils/pipe.js             system/pipe -a system true
 
-install utils/cat.js              util/cat -a description 'Concatenate array of strings'
-install utils/sort.js             util/sort -a description 'Sort array'
-install utils/head.js             util/head -a description 'Filter first K array elements and discard rest'
-install utils/date.js             util/date -a description 'Get current date and time'
+install utils/cat.js              util/cat -a description 'Concatenates input into a string' \
+                                           -a parameters '[ { "name": "lines", "required": true, "type": "array", "description": "An array of strings or numbers" } ]' \
+                                           -a sampleInput '{ "lines": [4, 2, 3] }' \
+                                           -a sampleOutput '{ "lines": [4, 2, 3], "payload": "4\n2\n3" }'
+
+install utils/sort.js             util/sort -a description 'Sorts an array' \
+                                           -a parameters '[ { "name": "lines", "required": true, "type": "array", "description": "An array of strings" } ]' \
+                                           -a sampleInput '{ "lines": [4, 2, 3] }' \
+                                           -a sampleOutput '{ "lines": [2, 3, 4], "length": 3 }'
+
+install utils/head.js             util/head -a description 'Extract prefix of an array' \
+                                           -a parameters '[ { "name": "lines", "required": true, "type": "array", "description": "An array of strings" }, { "name": "num", "required": false, "type": "integer", "description": "The length of the prefix" }]' \
+                                           -a sampleInput '{ "lines": [4, 2, 3], num: 2 }' \
+                                           -a sampleOutput '{ "lines": [4, 2], "num": 2 }'
+
+install utils/date.js             util/date -a description 'Current date and time' \
+                                           -a sampleOutput '{ "date": "2016-03-22T00:59:55.961Z" }'
+
+install samples/hello.js          samples/helloWorld -a description 'Demonstrates logging facilities' -a parameters '[{"name": "payload", "required":false, "description":"The string to be included in the log record"}]' \
+                                           -a sampleInput '{ "payload": "Cat" }' \
+                                           -a sampleOutput '{ }' \
+                                           -a sampleLogOutput '2016-03-22T01:02:26.387624916Z stdout: hello Cat!'
+
+install samples/greeting.js       samples/greeting -a description 'Returns a friendly greeting' -a parameters '[{"name": "name", "required":false}, {"name": "place", "required":false, "description":"The string to be included in the return value"}]' \
+                                           -a sampleInput '{ "name": "Cat", "place": "Narrowsburg" }' \
+                                           -a sampleOutput '{ "name": "Hello, Cat from Narrowsburg!" }' \
```

Comment:
The code says the output is not in a property `name` but instead `payload` (see [here](https://github.com/openwhisk/openwhisk/blob/master/catalog/samples/greeting.js)). Only one way to enforce the two are consistent: add tests that read the action annotations and use that to generate the input arguments and verify the output.